src/setMiddleC.js

```diff
@@ -26,20 +27,22 @@ function transposeOctave(initialOctave) {
 /**
  * Takes a noteObj and transposes the note into the octave given by transposition 
  * @param {String/Array} noteArg		The Array/String contaning the note(s)
+ * @param {Integer} octave Optional octave to transpose to  
  * @return {String(s)} 	The correctly transposed note(s)
  */
-function transposeNote(noteArg) {
+function transposeNote(noteArg, octave) {
```

Comment:
Since we have mandated Node 4+, we can use `const transposeNote = (noteArg, octave) => {`